src/modules/transaction/manager/transactionDetailViewManager.js

```diff
@@ -4,18 +4,14 @@ import { compose } from 'redux';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router-dom';
 import { withTranslation } from 'react-i18next';
-import grid from 'flexboxgrid/dist/flexboxgrid.css';
 import { selectActiveTokenAccount, selectActiveToken } from 'src/redux/selectors';
 import { getTransaction } from '@transaction/api';
-import Dialog from '@theme/dialog/dialog';
 import withData from 'src/utils/withData';
 import { parseSearchParams } from 'src/utils/searchParams';
 import TransactionDetails from '../components/TransactionDetailsView';
 
 const WrappedInDialog = (props) => (
```

Comment:
@eniolam1000752 Since the Dialog has been removed, this should be renamed appropriately